.github/workflows/ci_cd.yml

```diff
@@ -0,0 +1,56 @@
+name: Zulip Desktop CI
+
+on: [push]
+
+jobs:
+  test:
+    name: Test on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: 
+        - ubuntu-latest
+        - windows-latest
+#         - macOS-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setting up Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '*'
+
+    - name: Setup npm cache
+      id: npm-cache
+      run: |
+         echo "::set-output name=dir::$(npm config get cache)"
+         
+    - uses: actions/cache@v1
+      with:
+        path: ${{ steps.npm-cache.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+          
+    - name: Installing npm modules
+      run: npm install
+
+    - name: Running Tests
+      run: npm run test
+
+    - name: Building app installers
+      uses: samuelmeuli/action-electron-builder@v1.3.0
```

Comment:
Two of the major things I missed in the previous review:
  - We need to add a `build` npm script that runs `tsc`. We need to do this in order for this action to build typescript files before creating installers. Currently, the installers created do not run because of this.
  - In `package.json` we need to add `publish` property under `build` and set it to `never` so the release is not automatically pushed to GitHub. See https://www.electron.build/configuration/publish.

Reply:
@adityamudgil2505 I don't think you addressed these two issues yet in your latest push.

Reply:
@priyank-p I added a new script in package.json file.
`"build": "tsc --publish never"` 
and run this script in .yml file
```
- name: Running Tests
       run: |
         npm run test
         npm run build
```